content/pt-br/_index.html

```diff
@@ -41,14 +41,13 @@
 <h2>Os desafios da migração de mais de 150 micro serviços para o Kubernetes</h2>
         <p>Sarah Wells, Diretora Técnica de Operações e Confiabilidade do Financial Times</p>
         <button id="desktopShowVideoButton" onclick="kub.showVideo()">Assista Video</button>
-        <br>
-        <br>
-        <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/" id="desktopKCButton">KubeCon + CloudNativeCon na Europa de 19 a 22 de março de 2024</a>
-        <br>
-        <br>
-        <br>
-        <br>
-        <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america-2024/" button id="desktopKCButton">KubeCon + CloudNativeCon na América do Norte de 12 a 15 de novembro de 2024</a>
+
+<h3>Participe dos próximos eventos KubeCon + CloudNativeCon</h3>
+        <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/" class="desktopKCButton"><strong>Europe</strong> (Londres, abril 1-4)</a>
+        <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-china/" class="desktopKCButton"><strong>China</strong> (Hong Kong, junho 10-11)</a>
+        <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-japan/" class="desktopKCButton"><strong>Japan</strong> (Tóquio, junho 16-17)</a>
+        <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-india/" class="desktopKCButton"><strong>India</strong> (Haiderabade, agosto 6-7)</a>
+        <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america-2025/" class="desktopKCButton"><strong>North America</strong> (Atlanta, nov. 10-13)</a>
```

Comment:
Some minor bits:
* In Portuguese, we write the numerical days, followed by a preposition ("de"), then followed by the month name.
* It seems the countries/continents names were missing translations.

```suggestion
<h3>Participe dos próximos eventos KubeCon + CloudNativeCon</h3>
        <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/" class="desktopKCButton"><strong>Europa</strong> (Londres, 1-4 de abril)</a>
        <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-china/" class="desktopKCButton"><strong>China</strong> (Hong Kong, 10-11 de junho)</a>
        <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-japan/" class="desktopKCButton"><strong>Japão</strong> (Tóquio, 16-17 de junho)</a>
        <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-india/" class="desktopKCButton"><strong>Índia</strong> (Haiderabade, 6-7 de agosto)</a>
        <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america-2025/" class="desktopKCButton"><strong>América do Norte</strong> (Atlanta, 10-13 de novembro)</a>
```

Reply:
@stormqueen1990 thanks a lot for your help! I applied (and force-pushed) the correct dates. However, I'd like to keep the countries/continents' names intact, since I consider them a part of the official event name. E.g., it's _KubeCon + CloudNativeCon Europe_ (as it's written on the conference website), etc. Thus, it was intentional and the same was done in other localisations (where I propagated this update) :pray: